docs/using/examples/example.py

```diff
@@ -0,0 +1,60 @@
+import httplib
+import os
+import ssl
+import tempfile
+
+# NOTE - to run this example, you need at least python 2.7.9
```

Comment:
:question: Maybe instead of noting this, let the code error. Or make it error with a "python version not sufficient" type error.